aspnetcore/migration/21-to-22.md

```diff
@@ -29,6 +29,16 @@ Projects targeting .NET Framework may continue to use the TFM of a version great
 <TargetFramework>net461</TargetFramework>
 ```
 
+## IIS in-process hosting model
+
+To adopt the [in-process hosting model for IIS](xref:fundamentals/servers/aspnet-core-module#in-process-hosting-model), add the `<AspNetCoreHostingModel>` property with a value of `InProcess` to a `<PropertyGroup>` in the project file:
+
+```xml
+<AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+```
+
+For more information, see <xref:host-and-deploy/aspnet-core-module#hosting-model>.
```

Comment:
@shirhatti  we should potentially cover the Directory.GetCurrentDirectory() stuff here.

Reply:
We can cross-link to https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/aspnet-core-module?view=aspnetcore-2.2#hosting-model that covers the differences between the two hosting model